docker-start.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+[ -z "$UID" ] && UID=0
+[ -z "$GID" ] && GID=0
+
+# echo >> /etc/xxx and not adduser/addgroup because adduser/addgroup won't work if uid/gid already exists
+echo -e "xuser:x:${UID}:${GID}:xuser:/home/xuser:/bin/ash\n" >> /etc/passwd
+echo -e "xgroup:x:${GID}:xuser\n" >> /etc/group
+
+# it's better to do that (mkdir and chown) here than in the Dockerfile because it will be executed even on volumes if mounted
+mkdir -p /droppy-data 
+mkdir -p /droppy-data/config
+mkdir -p /droppy-data/files
```

Comment:
As I tried to explain in my PR note, I changed `config` to `/droppy-data/config` and `files` to `/droppy-data/files`. The idea is to let the user choose whether to bind only one folder (here `/droppy-data`) and let the the droppy cocker container handle how to store data in the volume, or to let the user fine tune where to store the config and where to store the config on the host filesystem (by mounting `/droppy-data/config`  and `/droppy-data/files`)

But I expected you to not like this. I'll provide you with another PR without those folders, you'll choose which one you want to merge.